islandora_solr_config/includes/rss_results.inc

```diff
@@ -191,7 +190,85 @@ class IslandoraSolrResultsRSS extends IslandoraSolrResults {
     $result['url'] = $base_url;
     $result['description'] = t('Aggregated search results of: @query', array('@query' => $query));
     $result['langcode'] = NULL;
+    $feed_url = request_uri();
+    $page_params = drupal_get_query_parameters();
+
+    if (!isset($page_params['page'])) {
+      $page_number = '0';
+    }
+    else {
+      $page_number = $page_params['page'];
+    }
+    if (isset($page_params['page'])) {
+      $first_params = $page_params;
+      unset($first_params['page']);
+      $link_first = url(
+        request_path(),
+        array(
+          'absolute' => TRUE,
+          'query' => $first_params,
+        )
+      );
+    }
+    else {
+      $link_first = $base_url . $feed_url;
+    }
+
+    $page_next = $page_number + 1;
+    if ($page_number > 0) {
+      $page_prev = $page_number - 1;
+    }
+
+    $next_page = array(
+      'query' => array(
+        'page' => $page_next,
+      )+$page_params,
+    );
+
+    // Only set Next link if there is a next page.
+    $rss_limit = variable_get('islandora_solr_config_rss_limit', 50);
```

Comment:
Is this variable defined here? Maybe some code isn't checked in to create this?

Reply:
I'm not sure what you mean?

Reply:
Where does this variable get set. A quick grep in the code base does not find it anywhere.

Reply:
You mean this? https://github.com/Islandora/islandora_solr_search/blob/97a6c0accb622ff9d7deb3dbbdee9e6ff38ef927/islandora_solr_config/includes/admin.inc#L148

Reply:
Ugh. My bad. I guess I grepped an old copy of the codebase. Sorry about that.